src/app/view/components/dot-listing-data-table/dot-listing-data-table.component.html

```diff
@@ -10,6 +10,9 @@
     />
     <ng-content></ng-content>
 </dot-action-header>
+
+<dot-empty-state [rows]="10" [cols]="[60, 50, 60, 30, 40]"></dot-empty-state>
```

Comment:
This should only show if there is no templates.

Reply:
This is WIP and only a draft PR.